sky/engine/BUILD.gn

```diff
@@ -35,6 +35,10 @@ config("non_test_config") {
 }
 
 group("engine") {
+  public_configs = [
```

Comment:
Oops, didn't think of that. Though you only meant to expose just the include dirs.